Base/usr/share/man/man1/ls.md

```diff
@@ -27,6 +27,7 @@ If no *path* argument is provided the current working directory is used.
 * `-r`, `--reverse`: Reverse sort order
 * `-G`: Use pretty colors
 * `-i`, `--inode`: Show inode ids
+* `-I`, `--raw-inode`: Show raw inode ids if possible (see Notes to understand when this will not work)
```

Comment:
Can't we warn the user when this is the case? Because we know when raw inode values were requested, and when we're using `readdir` vs `lstat` right?